datamol/reactions/_reactions.py

```diff
@@ -171,6 +174,9 @@ def select_reaction_output(
 
     Args:
         product: All the products from a reaction.
+        product_index: Index of the product to select.
+            Examples: A.B -> C.D. The indexes of products are 0 and 1.
+            Both C and D will be returned if index is None or product index is to [0, 1].
         single_output: Whether return a single output from a reaction.
```

Comment:
Should `single_output` be renamed to `single_product` to be more consistent with the other args?

Also I would add plurals to `Index of the product to select.` -> `Indices of the product(s) to select.`